armi/physics/fuelCycle/tests/test_fuelHandlers.py

```diff
@@ -122,7 +127,40 @@ def tearDown(self):
         self.directoryChanger.close()
 
 
+def getInterfaceMocked(self, name=None, function=None):
+    if name != "latticePhysics":
+        return self.getInterfaceBackup(name, function)
+
+    i = self.getInterfaceBackup("latticePhysics")
+    i.interactBOC = lambda: None
+    return i
+
+
+class MockLatticePhysicsInterface(LatticePhysicsInterface):
```

Comment:
Ditto about the docstring.

Reply:
https://github.com/terrapower/armi/pull/1308/commits/2eeaa34bdd0c57dbb82cccbbf3d186859cecdcfb